src/realtime.def.ts

```diff
@@ -55,7 +55,7 @@ export interface Vehicle {
 
 export interface TripUpdate {
   trip: RealTimeTrip;
-  stop_time_update?: StopTimeUpdate;
+  stop_time_update?: StopTimeUpdate[];
```

Comment:
Might want to rename this to `stop_time_updates` but that'd be a breaking change.

Reply:
We don't choose the field names, they're defined in [the GTFS spec](https://gtfs.org/realtime/reference/#message-tripupdate)